deny.toml

```diff
@@ -228,10 +228,10 @@ ignore = [
     "RUSTSEC-2021-0145",
     # Consider `encoding_rs` instead of `encoding` (unmaintained)
     "RUSTSEC-2021-0153",
-    # abomonation transmutes &T to and from &[u8] without sufficient constraints
-    "RUSTSEC-2021-0120",
```

Comment:
Why did we remove this one?

Reply:
We have upgraded the abomonation version to a version where Moritz fixed this advisory.